features/buyers/accounts/edit.feature

```diff
@@ -34,3 +34,62 @@ Feature: Audience > Accounts > Edit
     And should see the following table:
       | Group/Org. |
       | Pepe       |
+
+  @javascript
+  Scenario: Provider should see all fields defined for account
```

Comment:
Please don't move these features out of /old unless they're reworked. It's better (the lesser evil) to update/add new ones in the old file

Reply:
1. Why? There are bad scenarios that are not in `old`, aren't there? So we can't use it as a way to track bad scenarios (i.e. "if it's not in `old` it should be fine")

2. It has been refactored slightly and I don't see a reason why it shouldn't be moved to where it belongs (in the "old" file it was misplaced, IMO)

3. The new scenario, `Scenario: Edit fields with choices` also didn't belong to the "old" file, it belongs here. Why would I put it under "old"? :thinking:

Reply:
> 1. Why? There are bad scenarios that are not in `old`, aren't there? So we can't use it as a way to track bad scenarios (i.e. "if it's not in `old` it should be fine")

Even though not all old scenarios are under the /old folder, we don't have a better way to keep track of them.

> 2. It has been refactored slightly and I don't see a reason why it shouldn't be moved to where it belongs (in the "old" file it was misplaced, IMO)

Slight refactors are certainly welcome but as it has been discussed we're not using "I ..." steps anymore and it also uses other old steps such as: `Vat rate of buyer...`, `current domain is + log in`, `should see error in fields`. These steps even though valid, are old and still to be reviewed/improved.

> 3. The new scenario, `Scenario: Edit fields with choices` also didn't belong to the "old" file, it belongs here. Why would I put it under "old"? 🤔

Because it uses the same "old" steps so it's more convenient for me to have them under old. That way it will be easier for me to keep updating all cucumbers.

Reply:
Let's better move on, I don't want to block a PR based on mere suggestions. If there is room for improvements we can always do it another time.

Reply:
Addressed in https://github.com/3scale/porta/pull/3685/commits/4188fd6a2c385e7854dbc3e7d9ece8f7e8d6c2d2